src/selectors/entities/users.ts

```diff
@@ -306,6 +306,16 @@ export const getProfilesInTeam: (state: GlobalState, teamId: $ID<Team>) => Array
     },
 );
 
+export const getProfilesNotInTeam: (state: GlobalState, teamId: $ID<Team>) => Array<UserProfile> = createSelector(
+    getUsers,
+    getUserIdsNotInTeams,
+    (state: GlobalState, teamId: string) => teamId,
+    (state: GlobalState, teamId, filters) => filters,
```

Comment:
I guess I'm a little confused where `filters` is coming from? I don't quite understand where we get filters from. Also, in the same way we are doing above, should we do the following? 
```suggestion
    (state: GlobalState, teamId: string, filters: {object or whatever suppose be}) => filters,
```

Reply:
Thanks for quick fix!